backend/session/tests/test_views.py

```diff
@@ -45,3 +49,30 @@ def test_create_without_playlist(self):
             experiment=self.experiment, participant=self.participant)
         assert new_session
         assert new_session.playlist == self.playlist1
+
+    def test_next_round(self):
+        session = Session.objects.create(
+            experiment=self.experiment, participant=self.participant)
+        response = self.client.get(
+            '/session/{}/next_round/'.format(session.id))
```

Comment:
```suggestion
            f'/session/{session.id}/next_round/'
```

Sorry, my pet peeve :-)